packages/stryker-jasmine-runner/src/JasmineTestRunner.ts

```diff
@@ -0,0 +1,70 @@
+import { EOL } from 'os';
+import { TestRunner, RunResult, TestResult, RunStatus } from 'stryker-api/test_runner';
+import { StrykerOptions } from 'stryker-api/core';
+import { Jasmine, toStrykerTestResult, evalGlobal } from './helpers';
+
+export default class JasmineTestRunner implements TestRunner {
+
+  private jasmineConfigFile: string | undefined;
+  private fileNames: ReadonlyArray<string>;
+  private Date: { new(): Date } = Date; // take Date prototype now we still can (user might choose to mock it away)
+
+  constructor({ fileNames, strykerOptions: { jasmineConfigFile } }: { fileNames: ReadonlyArray<string>, strykerOptions: StrykerOptions }) {
```

Comment:
To me, this code is more readable:
```typescript
constructor(private fileNames: ReadonlyArray<string>, strykerOptions: StrykerOptions) {
    this.jasmineConfigFile = strykerOptions.jasmineConfigFile;
  }
```

Reply:
True, but defining it like this makes it much easier to test.

We don't need to specify all required stryker options, instead we can focus on only the stuff that's actually used.

For example:

```js
new JasmineTestRunner({
        fileNames: [
           //...
        ], strykerOptions: { jasmineConfigFile: 'spec/support/jasmine.json' }
});
```

Reply:
What could the impact of this notation be if we decide to refactor the StrykerOptions? Would our editors understand that this code is also using the StrykerOptions?